src/re_frame_datatable/subs.cljc

```diff
@@ -15,6 +15,23 @@
  (fn [db [_ db-id]]
    (db/options db db-id)))
 
+(defn- apply-case-sensitivity
+  "If case-insensitivity param is set to true lowercase passed string"
+  [s case-insensitive]
+  (if case-insensitive
+    (clojure.string/lower-case s)
```

Comment:
You could make this more generic by adding a `:filter-fn` or `:filter-tx` configuration parameter that allows you to specify the filter function/transducer.

Reply:
filtering-fn added as a option for filtering. Usage and default behaviour explained in a subsection of filtering in docs. Defaults to case sensitive filtering.